PWE.html

```diff
@@ -30,61 +32,118 @@
+</p>
+<p>Individuals are encouraged to try and resolve issues themselves in the first instance. However, if individuals do not feel comfortable or safe doing so, they can get help through the ombudsperson, facilitators and/or mediation. 
+</p>
+<p>Depending on the severity of the situation, it may not be appropriate to attempt to resolve the issue informally (for example, threats of violence, assault or sexual assault). Individuals should immediately report the incident whenever they feel unsafe or threatened by the behaviour or actions of other W3C participants.
+</p>
+<p>Everyone involved in the dispute resolution process is expected to:</p>
+<ul>
+     <li> show respect for others</li>
+    <li> work together to resolve the complaint</li>
+<li> maintain confidentiality</li>
+</ul>
+
+        <p>A disciplinary investigation can be started where one of the following apply:</p>
+<ul><li>it’s not appropriate to begin with mediation </li>
+<li> all other mediation or resolution attempts have not worked</li>
+</ul>
+
+         <section id="procedures">
+            <h2>Incident Resolution Procedures</h2>
+
+            <p>When an incident arises it is important for everyone involved to know what will happen and what resources are available to them. Any time a Code of Conduct violation or complaint occurs it is important that those involved, especially the party/parties affected by the incident, feel supported and respected.</p>
+
+            <p>How an incident is handled will depend on a number of factors including the type of incident, severity, frequency, and contributing factors (for example, has anyone been involved in incidents before, is there a pattern?).</p>
+
+            <p>The procedure for addressing an incident will include some or all of the following steps:</p>
+
+            <ol>
+                <li>Incident is addressed directly with the parties involved, if appropriate.</li>
+                <li>Incident is reported.
+                    <ul>
+                        <li>Reporting can take on many forms including reporting the issue to a chair, ombudsperson, through an anonymous report, or a member of the team.</li>
+                    </ul>
+                </li>
+                <li>Incident is discussed, impact is shared, and first resolution is proposed.
+                    <ul>
+                        <li>For incidents that have a clear path to resolution and low severity (for example, a harsh criticism in a meeting or inappropriate language), the focus should be on coming to a shared understanding of what happened and agreeing to a path to resolution (for example, an apology or editing a comment or minutes).</li>
+                    </ul>  
+                </li>
+                <li>
+                    ...
+                </li>
+            </ol>
+        </section>
+
+        <section id="ombuds">
+            <h2>The Role of the Ombuds</h2>

```

Comment:
I think this section (Role of the Ombuds) should be before the Incident Resolution Procedures, and the Procedures should be a sequential or tiered list of steps of increasing severity.